pages/apis/rest_api/agents.md

```diff
@@ -148,8 +148,8 @@ Success response: `200 OK`
 Instruct an agent to stop accepting new build jobs and shut itself down.
 
 ```bash
-curl -H "Authorization: Bearer $TOKEN" \
-  -X PUT "https://api.buildkite.com/v2/organizations/{org.slug}/agents/{id}/stop" \
+curl -H "Authorization: Bearer $TOKEN" -X PUT \
+  "https://api.buildkite.com/v2/organizations/{org.slug}/agents/{id}/stop" \
```

Comment:
```suggestion
curl -H "Authorization: Bearer $TOKEN" \
  -X PUT "https://api.buildkite.com/v2/organizations/{org.slug}/agents/{id}/stop" \
```

Reply:
Thanks @docwhat ,
Is there a reason why you want to move the `-X PUT` component from the 2nd to the 1st line?
The `PUT` part of this component has been there in our docs for a while.